org.adempiere.base/src/org/compiere/model/MTableIndex.java

```diff
@@ -149,41 +172,21 @@ private String createDDL()
 		StringBuilder sql = null;
 		if (!isCreateConstraint())
 		{
-			sql = new StringBuilder("CREATE ");
+			sql = new StringBuilder("CREATE");
 			if (isUnique())
-				sql.append ("UNIQUE ");
-			sql.append("INDEX ").append (getName())
+				sql.append (" UNIQUE");
+			sql.append(" INDEX ").append (getName())

```

Comment:
the white space changes for CREATE, UNIQUE AND INDEX is unnecessary.

Reply:
The sql string does not contain any double spaces. The only redundant space is before the opening parenthesis of the columnn list. I did not remove it because it enhances readability for when debugging.

Reply:
you confuse me here, where did I mention anything on "double spaces" here ? I'm just those changes above is not needed at all.

Reply:
IC.  done.